src/clients/UBAClient/UBAClientBase.ts

```diff
@@ -246,78 +269,37 @@ export class BaseUBAClient extends BaseAbstractClient {
+      tokenSymbol,
+      amount,
+      hubPoolBlockNumber,
+      depositChainId,
+      UBAActionType.Deposit
+    );
     return { lpFee, depositBalancingFee, systemFee: lpFee.add(depositBalancingFee) };
   }
 
-  /**
-   * Compute the entire UBA fee in the context of the UBA system for a given amount. The UBA fee is comprised of 7 return variables.
-   * @param depositChain The chainId of the deposit
-   * @param refundChain The chainId of the refund
-   * @param tokenSymbol The token to get the relayer fee for
-   * @param hubPoolTokenAddress The token address of the token on the hub pool chain
-   * @param amount The amount to get the relayer fee for
-   * @param hubPoolBlockNumber The block number to get the relayer fee for
-   * @param overrides The overrides to use for the LP fee calculation
-   * @returns The UBA fee for the given token on the given chainId at the given block number
-   */
-  public getUBAFee(
-    depositChain: number,
-    refundChain: number,
-    tokenSymbol: string,
-    amount: BigNumber,
-    hubPoolBlockNumber: number,
-    overrides?: UBALPFeeOverride
-  ): { relayerBalancingFee: BigNumber } & SystemFeeResult {
-    const [relayerFee, systemFee] = [
-      this.computeBalancingFee(tokenSymbol, amount, hubPoolBlockNumber, refundChain, UBAActionType.Refund),
-      this.computeSystemFee(depositChain, refundChain, tokenSymbol, amount, hubPoolBlockNumber, overrides),
-    ];
-    return {
-      relayerBalancingFee: relayerFee.balancingFee,
-      ...systemFee,
-    };
-  }
-
-  /**
-   * Compute the entire UBA fee in the context of the UBA system for a given amount. The UBA fee is comprised of 7 return variables. This function is used to compute the UBA fee for a given amount on multiple refund chains.
-   * The function returns a record of the UBA fee for each refund chain that is not too low.
-   * @param depositChain The chainId of the deposit
-   * @param refundChainCandidates The chainIds of the refund candidates
-   * @param tokenSymbol The token to get the relayer fee for
-   * @param amount The amount to get the relayer fee for
-   * @param hubPoolBlockNumber The block number to get the relayer fee for
-   * @param overrides The overrides to use for the LP fee calculation
-   * @returns A record of the UBA fee for each refund chain that is not too low
-   */
-  public getUBAFeeFromCandidates(
-    depositChain: number,
-    refundChainCandidates: number[],
-    tokenSymbol: string,
-    amount: BigNumber,
-    hubPoolBlockNumber: number,
-    overrides?: UBALPFeeOverride
-  ): Record<number, RelayerFeeResult & SystemFeeResult> {
-    const transformation = refundChainCandidates.map((refundChain) => {
-      return [
-        refundChain,
-        this.getUBAFee(depositChain, refundChain, tokenSymbol, amount, hubPoolBlockNumber, overrides),
-      ] as [number, RelayerFeeResult & SystemFeeResult];
-    });
-    return Object.fromEntries(transformation.filter(([, result]) => !result.amountTooLow));
-  }
```

Comment:
Is this no longer needed?

Reply:
I didn't see how this would be used, how would it? I also don't see a point to the `getUBAFee` function. Why not just call `computeBalancingFee ` combined with  `computeSystemFee`?

Reply:
Fair enough!